google-cloud-bigquery/acceptance/bigquery/table_test.rb

```diff
@@ -253,6 +253,36 @@
     query_job.data.class.must_equal Google::Cloud::Bigquery::Data
     query_job.data.total.wont_be :nil?
 
+    # Query Job - Statistics Query Plan
+    query_job.query_plan.wont_be_nil
+    query_job.query_plan.must_be_kind_of Array
+    query_job.query_plan.wont_be :empty?
+    stage = query_job.query_plan.first
```

Comment:
@blowmage Are you OK with `#query_plan` returning an array, even though it sounds like a singular noun? Pretty misleading if you ask me, but I'm just sticking to the underlying API.